contracts/teller-network/Fees.sol

```diff
@@ -36,45 +34,66 @@ contract Fees is Ownable {
      * @dev Can only be called by the owner of the contract
      *      TODO: if the contract will be changed to remove ownership, remove this function
      */
-    function setFeeDestinationAddress(address _addr) public onlyOwner {
+    function setFeeDestinationAddress(address payable _addr) public onlyOwner {
         feeDestination = _addr;
         emit FeeDestinationChanged(_addr);
     }
 
     /**
      * @notice Set Fee Amount
-     * @param _amount New Amount
+     * @param _feeMilliPercent New millipercent
      * @dev Can only be called by the owner of the contract
      *      TODO: if the contract will be changed to remove ownership, remove this function
      */
-    function setFeeAmount(uint _amount) public onlyOwner {
-        feeAmount = _amount;
-        emit FeeAmountChanged(_amount);
+    function setFeeAmount(uint _feeMilliPercent) public onlyOwner {
+        feeMilliPercent = _feeMilliPercent;
+        emit FeeMilliPercentChanged(_feeMilliPercent);
     }
 
     /**
      * @notice Withdraw fees by sending them to the fee destination address
      */
-    function withdrawFees() public {
-        uint fees = feeBalance;
-        feeBalance = 0;
-        require(feeToken.transfer(feeDestination, fees), "Error transfering fees");
-        emit FeesWithdrawn(fees);
+    function withdrawFees(address _tokenAddress) public {
```

Comment:
Are we still going to hold the fees in the escrow contract until this function is called? or should we send the fees immediatly to the stake contract?

Reply:
I didn't code the later part of the fees, ie I didn't do the part where we send to the arbiter or the staking pool yet because I wanted to get the first part out before it got more complex.